utils/spacewalk-utils.spec

```diff
@@ -95,9 +94,9 @@ Requires:       salt
 Requires:       spacewalk-admin
 Requires:       spacewalk-backend
 %if 0%{?suse_version} >= 1320
```

Comment:
We already now that RHEL/CentOS8 is to be python3 based, maybe we should use this occasion to add it here as well?